packages/collaboration-manager/src/CollaborationManager.ts

```diff
@@ -11,16 +13,66 @@ export class CollaborationManager {
    */
   #model: EditorJSModel;
 
+  /**
+   * UndoRedoManager instance to manage undo/redo operations
+   */
+  #undoRedoManager: UndoRedoManager;
+
+  /**
+   * Flag to control whether events should be handled to avoid putting operations to the stack on undo/redo
```

Comment:
```suggestion
   * Flag to control whether events should be handled to avoid putting operations to the stack on undo/redo. Used for preventing operations infinity loop on undo/redo
```